shared/storage.ts

```diff
@@ -56,6 +59,64 @@ async function generateHmac(
 }
 
 class Storage {
+  static async writeToBackupFile(data): Promise<string> {
+    const fileName = getBackupFileName();
+    const isDirectoryExists = await FileStorage.exists(backupFilePath);
+    if (isDirectoryExists) {
+      await FileStorage.removeItem(backupFilePath);
+    }
+    await FileStorage.createDirectory(backupFilePath);
+    const path = getBackupFilePath(fileName);
+    await FileStorage.writeFile(path, JSON.stringify(data));
+    return fileName;
+  }
+
+  static async exportData(encryptionKey: string) {
+    const completeBackupData = {};
+    const dataFromDB: Record<string, any> = {};
+
+    const allKeysInDB = await MMKV.indexer.strings.getKeys();
+    const keysToBeExported = allKeysInDB.filter(key =>
+      key.includes('CACHE_FETCH_ISSUER_WELLKNOWN_CONFIG_'),
+    );
+    keysToBeExported.push(MY_VCS_STORE_KEY);
+
+    const encryptedDataPromises = keysToBeExported.map(key =>
+      MMKV.getItem(key),
+    );
+
+    Promise.all(encryptedDataPromises).then(encryptedDataList => {
+      keysToBeExported.forEach(async (key, index) => {
+        let encryptedData = encryptedDataList[index];
+        if (encryptedData != null) {
+          const decryptedData = await decryptJson(encryptionKey, encryptedData);
+          dataFromDB[key] = JSON.parse(decryptedData);
+        }
+      });
+    });
+
+    completeBackupData['dataFromDB'] = dataFromDB;
+    completeBackupData['VC_Records'] = {};
+
+    let vcKeys = allKeysInDB.filter(key => key.indexOf('VC_') === 0);
+    for (let ind in vcKeys) {
+      const key = vcKeys[ind];
+      const vc = await Storage.readVCFromFile(key);
+      const decryptedVCData = await decryptJson(encryptionKey, vc);
+      const deactivatedVC = this.deactivateVC(decryptedVCData);
+      completeBackupData['VC_Records'][key] = deactivatedVC;
+    }
+    return completeBackupData;
+  }
+
+  static deactivateVC(data: string) {
```

Comment:
Rename this function name
deactivateVC -> it gives message that we are deactivating VC. Instead of this use name like
`removeWalletBindingDataBeforeBackup`

Reply:
Done.